activemodel/lib/active_model/error.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+module ActiveModel
+  # Represents one single error
+  # @!attribute [r] base
```

Comment:
We don't use yarn, we use rdoc, so we need to follow the Rails documentation guideline.